docs/iana-application.md

```diff
@@ -0,0 +1,51 @@
+### Transport protocols
+- [x] TCP
+- [x] UDP
+- [ ] SCTP
+- [ ] DCCP
+
+### Service Name
+StatsD
+
+### Desired Port Number
+8125
+
+### Description
+Network daemon for collecting metrics
+
+### Reference
+StatsD is used to collect statistics such as counters and timers to measure system and application performance. StatsD receives metrics over UDP or TCP in a basic line protocl of the following format:
+
+`<metricname>:<value>|<type>`
+
+The metric name (or "bucket") is an identifier for the metric being sent. Values represent the metric value being pushed to statsd and should be of the specified "type". Statsd supports numerous mertric types such as counters, gagues, and timers. Each has it's own single character key which is used in the type section.
+
+Metrics are collected and flushed at a desired interval to the plugable backend for storage and analysis.
+
+### If UDP is requested, please explain how traffic is limited, and whether the protocol reacts to congestion.
+Statsd does not react to congestion itself, but does provide the means for creating statsd clusters to aid performance.
+
+### If UDP is requested, please indicate whether the service is solely for the discovery of hosts supporting this protocol.
+Statsd is not intended for service discovery.
+
+### Please explain how your protocol supports versioning.
+As new metric types are introduced they will be assigned their own "short code". Statsd is intended to be run internally and not exposed to the wider world and so ensuring the statsd version supports the desired metric type is the responsibility of the administrator.
+
+### If your request is for more than one transport, please explain in detail how the protocol differs over each transport.
+Statsd defaults to UDP but can be configured to use TCP to ensure delivery of packets. The protocol's contracts across TCP is the same as across UDP.
+
+### Please describe how your protocol supports security. Note that new services are expected to support security capabilities and to avoid insecure variants.
+Statsd can be run behind authentication services which can be errected to proxy traffic through to statsd if users wish to expose it across public networks such as the internet.
+
+The primary statsd server implementation maintains support for all current and LTS versions of nodejs. The protocol itself is small enough to enable the addition of authentication tokens should it fit the use case in the future via purely additive means.
```

Comment:
Should we add that a recommended mode of operation is to run statsd on localhost and rarely, if ever, expose to the wider internet?

Reply:
Yes I think that's a good idea, I'll whip up a few lines about that as well